vllm/engine/arg_utils.py

```diff
@@ -1630,7 +1630,10 @@ def _set_default_args_v1(self, usage_context: UsageContext) -> None:
             # This is only used to set default_max_num_batched_tokens
             device_name = "no-device"
 
-        if "h100" in device_name or "h200" in device_name:
+        large_mem_gpus = [
```

Comment:
can we just check the available memory directly?

Reply:
Good point. Updated the PR!